source/adapters/level_zero/adapter.cpp

```diff
@@ -544,7 +544,7 @@ ur_result_t adapterStateTeardown() {
       {"zeCommandListCreateImmediate", "zeCommandListCreate", "zeCommandListDestroy"},
       {"zeEventCreate",        "zeEventDestroy"},
       {"zeFenceCreate",        "zeFenceDestroy"},
-      {"zeImageCreate",        "zeImageDestroy"},
+      {"zeImageCreate","zeImageViewCreateExt"        "zeImageDestroy"},
```

Comment:
There should be a comma between "zeImageViewCreateExt" and "zeImageDestroy"

Reply:
yes, sorry about that, I will fix that immediately. typo when copying the change.